client/components/button/style.scss

```diff
@@ -102,7 +102,7 @@ button {
 }
 
 // Primary buttons
-.button.is-primary {
+@mixin button-is-primary {
```

Comment:
We shouldn't make a mixin out of this. What is the purpose? To switch a button style based on viewport? That seems like something we should do programatically like:

```
viewport.isMobile() ? <Button> : <Button primary>
```

Switching the CSS seems very fragile.